versioned_docs/version-5/xstate-vscode-extension.mdx

```diff
@@ -56,14 +56,14 @@ createMachine({});
 
 ## Machine layout persistence
 
-Upon opening an XState machine in the VSCode Editor, you may notice a long `@xstate-layout` comment inserted in the code just above the call to `createMachine()`.
+Upon opening an XState machine in the VSCode editor, you may notice a long `@xstate-layout` comment inserted in the code just above the call to `createMachine()`.
```

Comment:
Saw instances of "VS Code" vs "VSCode". 

From their website, it looks like the 2 choices are:

1. "Visual Studio Code"
2. "VS Code"